simexpal/base.py

```diff
@@ -1204,6 +1212,23 @@ def effective_thread_settings(self):
 			return s.thread_settings
 		return self.info.thread_settings
 
+	@property
+	def is_effective_exclusive(self):
+		s = None
+		for variant in self.variation:
+			vs = variant.is_exclusive
+			if vs is None:
+				continue
+			if s is not None and s.is_exclusive != vs:
+				raise RuntimeError("Exclusive flag for experiments '{}' overridden by multiple variants: {}".format(
+					self.name, [s.name, variant.name]
+				))
+			s = variant
+
+		if s is not None:
```

Comment:
The final check can be put into a single line and return statement:
```python
return s.is_exclusive if s is not None else self.info.is_exclusive
```

Reply:
Simplified it for the `effective_process_settings` and `effective_thread_settings` functions. (Since it is no longer needed for `is_exclusive`)